app/services/user-agent-service/index.js

```diff
@@ -13,9 +13,10 @@
 import * as userAgentService from './server';
 import { ProfileStorage } from './storage';
 import meta from './meta.json';
+import { logger } from './logging';
 
-process.on('uncaughtException', console.error);
-process.on('unhandledRejection', console.error);
+process.on('uncaughtException', (...args) => logger.error(...args));
+process.on('unhandledRejection', (...args) => logger.error(...args));
```

Comment:
Ditto here for the change. Why not bind in the logging module itself, if necessary?